sbt-crossproject/src/main/scala-sbt-1.0/sbt/StringUtilities.scala

```diff
@@ -0,0 +1,6 @@
+package sbt
```

Comment:
We really should not put anything in the `sbt` package. There's no excuse for this. Other plugins could be tempted to do the same, and then you have the same class defined twice on the classpath.

IIRC that method, `StringUtilities.nonEmpty`, was very short. I suggest to copy it and integrate it as private inside sbt-crossproject, and stop using the one coming from sbt.